GliaWidgets/SecureConversations/Welcome/Theme+SecureConversationsWelcome.swift

```diff
@@ -219,7 +219,7 @@ extension Theme {
                 error: error,
                 progressColor: color.primary,
                 errorProgressColor: color.systemNegative,
-                progressBackgroundColor: Color.lightGrey,
+                progressBackgroundColor: Color.baseNeutral,
```

Comment:
Do we need to use color from palette?
```suggestion
                progressBackgroundColor: color.baseNeutral,
```

Reply:
Why didn't we use it in the first place?

Reply:
https://github.com/salemove/ios-sdk-widgets/pull/777#discussion_r1331610131

Reply:
fixed [f8b8b60](https://github.com/salemove/ios-sdk-widgets/pull/777/commits/f8b8b600ed2c3cccef497d0c35a59dee80a4cfea)